app/assets/stylesheets/application.scss

```diff
@@ -1,9 +1,9 @@
-@charset "utf-8";
-
-@import "normalize-rails";
-
-@import "bourbon";
-@import "neat";
-
-@import "base/base";
-@import "refills/flashes";
+// @charset "utf-8";
```

Comment:
I removed the default styles as I will worry about styling the application later.